### PR TITLE
Add new zeit online mobile app

### PR DESCRIPTION
**Pull request by vanetix**
### Description:

This adds a new mobile app by ZEIT Media, which is relatively new, but is just now showing up in our logs. Please let me know if there's anything else that's needed.

- Link to announcement page: https://www.zeit.de/administratives/zeit-online-app-ios-android
- iOS app: https://apps.apple.com/us/app/zeit-online/id828889166
- Andriod app: https://play.google.com/store/apps/details?id=de.zeit.online&zonappclid=zona-promo

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [x] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

File: Tests/fixtures/mobile_apps.yml [hunk 1]
```diff
@@ -1949,3 +1949,35 @@
...
+  browser_family: Unknown
+-
+  user_agent: Mozilla/5.0 (Linux; Android 14; XQ-CC54 Build/65.2.A.2.179; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/128.0.6613.146 Mobile Safari/537.36 ZONApp/Android/2.2.7
+  os:
+    name: Android
+    version: "14"
+    platform: ""
+  client:
+    type: mobile app
+    name: zei ONLINE
```
Comment (Simbiat) [inline Tests/fixtures/mobile_apps.yml:1976]: is it missing a "t"? any reason "zei" is in lower case, and not upper one as in another example?

Comment (liviuconcioiu) [inline Tests/fixtures/mobile_apps.yml:1976]:
```suggestion
    name: ZEIT ONLINE
```

File: regexes/client/mobile_apps.yml [hunk 2]
```diff
@@ -2559,3 +2559,8 @@
 - regex: '(?!AlohaBrowser)([^/;]*)/(\d+\.[\d.]+) \((?:iPhone|iPad); (?:iOS|iPadOS) [0-9\.]+; Scale/[0-9\.]+\)'
   name: '$1'
   version: '$2'
+
+# ZEIT-Online app https://www.zeit.de/administratives/zeit-online-app-ios-android
+- regex: 'ZONApp/(:?iOS|Android)/([^;|$]+)'
+  name: 'ZEIT ONLINE'
+  version: '$2'
```
Comment (liviuconcioiu) [inline regexes/client/mobile_apps.yml:2566]:
```suggestion
- regex: 'ZONApp/(?:iOS|Android)/([\d\.]+)'
  name: 'ZEIT ONLINE'
  version: '$1'
```

File: Tests/fixtures/mobile_apps.yml [hunk 3]
```diff
@@ -1949,3 +1949,35 @@
...
+-
+  user_agent: Mozilla/5.0 (iPhone; CPU iPhone OS 17_7 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/15E148 ZONApp/iOS/2.5.6;build:3000000110
+  os:
+    name: iOS
+    version: "17.7"
+    platform: ""
+  client:
+    type: mobile app
+    name: ZEIT ONLINE
+    version: "2.5.6"
```
Comment (liviuconcioiu) [inline Tests/fixtures/mobile_apps.yml:1961]:
```suggestion
    version: 2.5.6
```

File: Tests/fixtures/mobile_apps.yml (hunk 1 above)
Comment (vanetix) [reply Tests/fixtures/mobile_apps.yml:1976]: Yes sorry about that. For some reason this had reverted to a point I was trying to confirm tests were updated and I didn't catch it.

**Comment by vanetix** [comment]
> You should also add these tests here Tests/Parser/Client/fixtures/mobile_app.yml.
> 
> ```yaml
> -
>   user_agent: Mozilla/5.0 (iPhone; CPU iPhone OS 17_7 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/15E148 ZONApp/iOS/2.5.6;build:3000000110
>   client:
>     type: mobile app
>     name: ZEIT ONLINE
>     version: 2.5.6
> -
>   user_agent: Mozilla/5.0 (Linux; Android 14; XQ-CC54 Build/65.2.A.2.179; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/128.0.6613.146 Mobile Safari/537.36 ZONApp/Android/2.2.7
>   client:
>     type: mobile app
>     name: ZEIT ONLINE
>     version: 2.2.7
> ```

Thanks for that, I didn't notice those tests prior, and thanks for all the feedback!

**Review by liviuconcioiu — APPROVED** [review]

**Pull request merged**